codegen/src/main/java/software/amazon/awssdk/codegen/poet/builder/AsyncClientBuilderClass.java

```diff
@@ -41,6 +42,7 @@ public AsyncClientBuilderClass(IntermediateModel model) {
         this.builderInterfaceName = ClassName.get(basePackage, model.getMetadata().getAsyncBuilderInterface());
         this.builderClassName = ClassName.get(basePackage, model.getMetadata().getAsyncBuilder());
         this.builderBaseClassName = ClassName.get(basePackage, model.getMetadata().getBaseBuilder());
+        // this.endpointRulesSpecUtils = new EndpointRulesSpecUtils(model);
```

Comment:
Is this commented-out code just temporary?

Reply:
Yeah breaking up the code into multiple PRs was a little awkward